src/pages/Result.js

```diff
@@ -1,7 +1,22 @@
 import React from "react";
+import Navbar from "../components/Navbar/Navbar";
+import ResultBody from "../components/Result-body/ResultBody";
+import Footer from "../components/Footer/Footer";
 
 function Result() {
-  return <div>This is results</div>;
+  return (
+    <div className="container">
+      <div className="nav">
+        <Navbar />
+      </div>
+      <div className="body-container-result">
+        <ResultBody />
+      </div>
+      <div className="footer">
+        <Footer />
+      </div>
+    </div>
```

Comment:
instead of repeating common elements like nav and footer, you can put them in  a layout

Reply:
I will look into it.